curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comment-your-javascript-code.english.md

```diff
@@ -16,7 +16,7 @@ Using <code>//</code> will tell JavaScript to ignore the remainder of the text o
 // This is an in-line comment.
 ```
 
-You can make a multi-line comment beginning with <code>/*</code> and ending with <code>*/</code>:
+You can make a multi-line comment beginning with <code>/\*</code> and ending with <code>\*/</code>:
```

Comment:
Actually, it would be better to use the following:
```suggestion
You can make a multi-line comment beginning with <code>/\*</code> and ending with <code>\*/</code>:
```

Reply:
Happy to make additional changes but I am not seeing the difference in your suggested change (and github won't let me commit the suggestion) - what did you have in mind?

Reply:
I have fixed my suggestion.